library/spit_results.py

```diff
@@ -493,7 +493,14 @@ def write_to_csv(self):
             # Write a CSV header if necessary
             file_size = os.path.getsize(normalized_path)
             if file_size == 0:
-                writer.writeheader()
+                # handle Python 2.6 not having writeheader method
+                if sys.version_info[1] > 6:
```

Comment:
Same here, it would be better to check for the major version since someone could be using on Python 3.5

Reply:
This may help:

On Python 2.7

```python
>>> sys.version_info >= (2, 7)
True
```

On python 2.6:

```python
>>> sys.version_info >= (2, 7)
False

>>> sys.version_info >= (2, 6)
True
```

Reply:
Good catch. Pushed a new commit for this.

Reply:
You can even compare micro version:

```python
>>> sys.version_info
(2, 6, 6, 'final', 0)
>>> sys.version_info >= (2, 6, 6)
True
>>> sys.version_info >= (2, 6, 5)
True
>>> sys.version_info >= (2, 6, 7)
False
```